src/pages/iou/MoneyRequestModal.js

```diff
@@ -0,0 +1,446 @@
+import _ from 'underscore';
+import React, {useState, useEffect, useRef} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import IOUAmountPage from './steps/IOUAmountPage';
+import IOUParticipantsPage from './steps/IOUParticipantsPage/IOUParticipantsPage';
+import IOUConfirmPage from './steps/IOUConfirmPage';
+import ModalHeader from './ModalHeader';
+import styles from '../../styles/styles';
+import * as IOU from '../../libs/actions/IOU';
+import Navigation from '../../libs/Navigation/Navigation';
+import ONYXKEYS from '../../ONYXKEYS';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import * as OptionsListUtils from '../../libs/OptionsListUtils';
+import FullScreenLoadingIndicator from '../../components/FullscreenLoadingIndicator';
+import AnimatedStep from '../../components/AnimatedStep';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import CONST from '../../CONST';
+import * as PersonalDetails from '../../libs/actions/PersonalDetails';
+import withCurrentUserPersonalDetails from '../../components/withCurrentUserPersonalDetails';
+import networkPropTypes from '../../components/networkPropTypes';
+import {withNetwork} from '../../components/OnyxProvider';
+import reportPropTypes from '../reportPropTypes';
+import * as ReportUtils from '../../libs/ReportUtils';
+import * as ReportScrollManager from '../../libs/ReportScrollManager';
+
+/**
+ * A modal used for requesting money, splitting bills or sending money.
+ */
+const propTypes = {
+    /** Whether the IOU is for a single request or a group bill split */
```

Comment:
nab, should we change some of these `IOU` to `Request`?

Reply:
Updated some of them to Request - didn't update all of them because they are still IOU reports and used like that throughout the code so didn't want it to cause more confusion